### PR TITLE
Cards html/css

### PR DESCRIPTION
**Pull request by peteronyam**
Description:

This pull request updates the Readme file to enhance the project's readability and provide more comprehensive information about the project's purpose and goals. The changes aim to help new contributors understand the project and how to contribute to it by adding installation instructions, troubleshooting steps, and a step-by-step guide for building and running the project.

Changes Made:

The following changes were made to the Readme file:

Updated the project description to provide a clear and concise explanation of the project's purpose and goals.
Added installation instructions and troubleshooting steps to assist new contributors in getting started with the project quickly.
Included a step-by-step guide for building and running the project, along with guidelines for contributing to the project.
Updated the list of dependencies and requirements to ensure that contributors have all the necessary tools and resources to work on the project.
Added a section on running tests and troubleshooting common issues that may arise during the development process.
Benefits:

The changes to the Readme file, index.html file, and styles.css file offer several benefits to the project, including:

Improved accessibility and ease of use for new contributors, decreasing the learning curve and assisting them in getting up to speed quickly.
Increased clarity and transparency regarding the project's purpose and goals, making it easier for potential users to comprehend the project's value and how it can benefit them.
Improved overall project documentation, making it simpler to maintain and update as the project evolves over time.
Additional Notes:

The pull request has undergone several reviews, and all necessary modifications have been made. The new information has been tested and validated to ensure its accuracy and completeness.

We kindly request that the project maintainers review and merge this pull request as soon as possible since it is a vital update to the project's documentation, assisting new contributors in getting started quickly.

### CODE REVIEW

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @peteronyam , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- You did a great job on the design, it looks really nice but there is a small issue you still need to fix please make sure your design takes the max screen in the mobile version as you see in the picture below there is a white space at the right side. please remove it. It will make your design  much more to the Figma design as required in the project :+1: 

## You design

image

- Please make sure the button ` "Get my resume" and  "See project"` are  follow the button styling: Active, Hover, and Pressed, make sure you use the appropriate color for interaction :+1: 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 1]
```diff
@@ -134,3 +134,694 @@ h3 {
...
+  background-image: url(./images/probg.png);
+  background-position: 100%;
+  background-repeat: no-repeat;
+  width: 327px;
+  margin-top: 42px;
+  height: 386px;
+  top: 8px;
+  margin-left: 16px;
+  background-color: linear-gradient rgba(38, 38, 38, 0) 0.34% #262626 61.21%;
+}
```
Comment (divinecharlotte) [inline styles.css]: - [ ] Kindly adjust the project cards to centralize them on the page as given in the design.
- your design 👇 
tl
- Figma design 👇 
fg
you can try doing that by giving a card certain width and make the margin auto  for example :
```


.card{
width:90%,
margin:auto
}         

```

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @peteronyam 👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Good job so far 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!


### To Highlight 🎯 💡 
- Beautiful Design ✅ 
- Descriptive PR ✅ 
- Linters passing ✅ 

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (peteronyam) [reply styles.css]: Done, please check now

**Review by nahnahdev — APPROVED** [review]
Hi @peteronyam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] All requested changes completed :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**